controllers/kafka.go

```diff
@@ -0,0 +1,48 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+	"zestream/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+/*
+	{
+	  "video": {
+	    "id": "string",
+	    "src": "string",
+	    "type": "mp4"
+	  }
+	}
+*/
+type Video struct {
+	ID   string `json:"id"`
+	Src  string `json:"src"`
+	Type string `json:"type"`
+}
+
+type Body struct {
+	Video Video `json:"video"`
+}
+
+func PublishMessage(c *gin.Context) {
+	var request Body
+	if err := c.ShouldBindJSON(&request); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	jsonBytes, err := json.Marshal(request)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	var message = utils.PublishMessage(utils.Config["KAFKA_URI"], string(jsonBytes), "video")
+	if message {
+		c.JSON(http.StatusCreated, gin.H{"status": "success"})
+		return
+	} else {
```

Comment:
@abhishekraj272 have you decided on the error response schema for API?
from me, it should be consistent. instead of status, it should be an error key for error response

Reply:
@akshaym-3255 I have returned "status": "success" only for successful publish of the message, other wise it will return "error": "<error_message>"